src/test/java/com/wikia/webdriver/common/dataprovider/ads/FandomAdsDataProvider.java

```diff
@@ -21,19 +21,9 @@
 
   private FandomAdsDataProvider() { }
 
-  @DataProvider
-  public static Object[][] fandomAds() {
-    return new Object[][]{
-        {
-            AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
-            "young-fans-guide-cinema-part-3"
-        },
-        {
-            AdsFandomTestTemplate.PAGE_TYPE_TOPIC,
-            "movies"
-        }
-    };
-  }
+  public static final String PAGE_NON_UAP_TOPIC = "movies";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)